### PR TITLE
Updated package management docs for nix flakes / devenv

### PR DESCRIPTION
**Pull request by mpscholten**
Updates docs for devenv + nix flakes

### CODE REVIEW

File: Guide/package-management.markdown [hunk 1]
```diff
@@ -6,63 +6,139 @@
 
 IHP is using the Nix Package Manager for managing its dependencies, such as Haskell packages, compilers, and even Postgres. You can find more about [the motivation on using nix on the IHP blog](https://ihp.digitallyinduced.com/blog/2020-07-22-why-ihp-is-using-nix.html).
 
+Internally IHP is using [devenv.sh](https://devenv.sh/) together with [nix flakes](https://devenv.sh/guides/using-with-flakes/).
+
 ## Using a Haskell Package
 
-To install a Haskell package from Hackage (the standard Haskell package registry), open the `default.nix` file and append the package name.
+To install a Haskell package from Hackage (the standard Haskell package registry), open the `flakes.nix` file and append the package name.
```
Comment (amitaibu) [inline Guide/package-management.markdown:13]: I think the file name is flake.nix

Comment (mpscholten) [reply Guide/package-management.markdown:13]: Fixed

Comment (amitaibu) [reply Guide/package-management.markdown:13]: I can still see `flakes.nix` here and below

```suggestion
To install a Haskell package from Hackage (the standard Haskell package registry), open the `flake.nix` file and append the package name.
```

Comment (mpscholten) [reply Guide/package-management.markdown]: sorry, fixed now 👍

**Pull request merged**